wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -129,6 +129,10 @@ public class UserClient: ZMManagedObject, UserClientType {
     public var e2eIdentityCertificate: E2eIdentityCertificate?
     public var mlsThumbPrint: String?
 
+    public var resolvedMLSThumbprint: String? {
+        e2eIdentityCertificate?.mlsThumbprint ?? mlsPublicKeys.ed25519
```

Comment:
This line is wrong now, because it was changed in my PR in the place you moved it from:
https://github.com/wireapp/wire-ios/pull/1110

After @KaterinaWire suggestion I just used `e2eIdentityCertificate?.mlsThumbprint` right in the places.

Reply:
We could tackle this in your PR. It is related @markusfassbender